Makefile

```diff
@@ -119,9 +120,11 @@ install-helm:
 
 .PHONY: e2e-local-bootstrap
 e2e-local-bootstrap:
-	kind create cluster --image kindest/node:v${KIND_K8S_VERSION} --config test/kind-config.yaml
+	kind create cluster --image kindest/node:v${KIND_K8S_VERSION}${KIND_K8S_SHA}
```

Comment:
Do we need to use a specific sha or is the k8s version sufficient?

Reply:
Since they upgraded to `0.9.0` and have released a Kind K8s version of 1.19.X, `1.18.2` doesn't require a SHA anymore to a specific image. They normally advise when using a new release to specify the docker image to be used